package.json

```diff
@@ -7,17 +7,19 @@
     "packages/**/*"
   ],
   "scripts": {
-    "new": "ts-node ./packages/scripts/src/new",
+    "new": "ts-node-transpile-only ./packages/scripts/src/new",
     "bundle": "ncc build",
     "clean": "tsc -b --clean  ./packages/tsconfig.json",
     "lint": "eslint --ext ts .",
     "lint:fix": "yarn lint --fix",
     "setup": "tsc -b ./packages/tsconfig.json ",
     "test": "jest",
-    "generate:docker-compose": "ts-node ./packages/scripts/src/docker-build",
-    "generate:oas": "ts-node ./packages/scripts/dist/scripts/src/docgen.js",
+    "generate:docker-compose": "ts-node-transpile-only ./packages/scripts/src/docker-build",
+    "generate:oas": "ts-node-transpile-only ./packages/scripts/dist/scripts/src/docgen.js",
     "generate:oas:all": "yarn generate:oas --all",
-    "generate:gha:matrix": "ts-node ./packages/scripts/src/gha",
+    "generate:gha:matrix": "ts-node-transpile-only ./packages/scripts/src/gha",
+    "generate:schemas": "ts-node-transpile-only ./packages/scripts/src/env-gen",
```

Comment:
Where do these come from? I don't see the files committed.

Reply:
ah yeah, removing this. the docker-compose script calls the schema generator internally